plsync/planetlab/model_test.py

```diff
@@ -20,6 +20,53 @@ def setUp(self):
         for hostname, node in self.sites[0]['nodes'].iteritems():
             self.experiments[0].add_node_address(node)
 
+    def test_site_ipv4(self):
+        expected_ip = '192.168.1.2'
+        site = self.sites[0]
+
+        ip = site.ipv4(index=2)
+
+        self.assertEqual(expected_ip, ip)
+
+    def test_pcu_hostname(self):
+        expcted_hostnames = [
```

Comment:
Can we add the E to expcted?  Removing it only saves one character, which doesn't seem worth having a misspelled word.  Not a big deal, though.